introduction/Organization.tex

```diff
@@ -34,11 +31,12 @@ \subsection{Organizing Committee --- oc@robocupathome.org}
 The \iaterm{Organizing Committee}{OC} is responsible for the organization of the competition. Members of the RoboCup@Home Organizing Committee for \YEAR:
 
 \begin{itemize}
-  \item Mauricio Matamoros (Universidad Nacional Autonoma de Mexico, Mexico)
+  \item[Chair] Mauricio Matamoros (University of Koblenz-Landau, Germany)
+  \item Alexander William Moriarty (Fetch Robotics, USA)
+  \item Fabrice Jumel (Université de Lyon; France)
   \item Francisco Javier Rodriguez Lera (University of Luxembourg, Luxembourg)
-  \item Raphael Memmesheimer (University of Koblenz, Germany)
-  \item Alexander William Moriarty (TBA, Bonn-Rhein-Sieg University; Germany)
   \item Jeffrey Too Chuan Tan (The University of Tokyo, Japan)
-%  \item Josemar Rodrigues de Souza (Bahia State University, Brazil)
-%  \item Ryuichi Ued (Chiba Institute of Technology, Japan)
+  \item Jesus Savage (Universidad Nacional Autónoma de México, Mexico)
+  \item Loy van Beek (Nobleo Technology, The Netherlands)
```

Comment:
Wait, what?

Reply:
I'll help out if needed, but want to mostly be a team member and do some coding during the tournaments.